### PR TITLE
add "alignLeft" and "alignRight" functions

### PR DESCRIPTION
**Pull request by zeitlinger**
Fixes https://github.com/grafana/loki/issues/9667

### CODE REVIEW

File: docs/sources/query/template_functions.md [hunk 1]
```diff
@@ -672,7 +698,7 @@ Examples:
 
 ```template
 {{ default "-" "" }} // output: -
-{{ default "-" "foo" }} // output: foo
```
Comment (zeitlinger) [inline docs/sources/query/template_functions.md:675]: I think this was wrong

**Comment by MichelHollands** [comment]
@zeitlinger Thanks for the contribution. Once you sign the CLA I'll approve and merge this.

**Comment by zeitlinger** [comment]
> @zeitlinger Thanks for the contribution. Once you sign the CLA I'll approve and merge this.

done :smile:

**Review by MichelHollands — APPROVED** [review]

**Comment by cyrille-leclerc** [comment]
Solution:

```
{{ alignLeft 5 "hello world"}}
{{ alignRight 5 "hello world"}}
```

**Pull request merged**